proto/vtctldata.proto

```diff
@@ -42,6 +43,114 @@ message ExecuteVtctlCommandResponse {
   logutil.Event event = 1;
 }
 
+message ChangeTabletTypeRequest {
+  topodata.TabletAlias tablet_alias = 1;
+  topodata.TabletType db_type = 2;
+  bool dry_run = 3;
+}
+
+message ChangeTabletTypeResponse {
+  topodata.Tablet before_tablet = 1;
+  topodata.Tablet after_tablet = 2;
+  bool was_dry_run = 3;
+}
+
+message CreateKeyspaceRequest {
+  // Name is the name of the keyspace.
+  string name = 1;
+  // Force proceeds with the request even if the keyspace already exists.
+  bool force = 2;
+  // AllowEmptyVSchema allows a keyspace to be created with no vschema.
+  bool allow_empty_v_schema = 3;
+
+  // ShardingColumnName specifies the column to use for sharding operations.
+  string sharding_column_name = 4;
+  // ShardingColumnType specifies the type of the column to use for sharding
+  // operations.
+  topodata.KeyspaceIdType sharding_column_type = 5;
+
+  // ServedFroms specifies a set of db_type:keyspace pairs used to serve
+  // traffic for the keyspace.
+  repeated topodata.Keyspace.ServedFrom served_froms = 6;
```

Comment:
On `served_froms` specifically, the docs list `served_from` 🤔 are they supposed to be the same?

Reply:
I think I can take this one -- the existing docs list the arg as `-served_from=tablettype1:ks1,tablettype2:ks2,...`

but served_from takes a list and gets mapped into ServedFroms for the actual call if more than one is provided:

```go
        if len(servedFrom) > 0 {
                for name, value := range servedFrom {
                        tt, err := parseServingTabletType3(name)
                        if err != nil {
                                return err
                        }
                        ki.ServedFroms = append(ki.ServedFroms, &topodatapb.Keyspace_ServedFrom{
                                TabletType: tt,
                                Keyspace:   value,
                        })
                }
        }
```

This is formalizing that CLI behavior into a request object.